File: fix_quotes.js
import fs from 'fs';

// Read the file
let content = fs.readFileSync('routes/mockInterview.js', 'utf8');

// Replace escaped quotes
content = content.replace(/\\\"/g, '"');

// Write back
fs.writeFileSync('routes/mockInterview.js', content, 'utf8');

console.log('✅ Fixed escaped quotes in mockInterview.js');

File: fix_quotes.py
import re

# Read the file
with open('routes/mockInterview.js', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace the escaped quotes
content = content.replace(r'\"', '"')

# Write back
with open('routes/mockInterview.js', 'w', encoding='utf-8') as f:
    f.write(content)

print("✅ Fixed escaped quotes in mockInterview.js")
